exercises/sum-of-multiples/description.md

```diff
@@ -1,4 +1,5 @@
-Given a number, find the sum of all the multiples of particular numbers up to but not including that number.
+Given a number, find the sum of all the multiples of particular numbers up to
```

Comment:
This is almost identically repeated at the end of the description. "particular numbers" also seems to be confusing language here.